pkg/util/parameterutil.go/get_max_len.go

```diff
@@ -1,4 +1,4 @@
-package proxy
+package parameterutil
```

Comment:
utils should be with a short name, or no one is willing to use it, -> `paramutil`, but this should be in `typeutil` package